File: content/problems/0997-find-the-town-judge/solutions/net-trust-degree/Solution.java
class Solution {
    public int findJudge(int n,int[][] trust){
        int[] score=new int[n+1];
        for(int[] relation:trust){
            score[relation[0]]--;
            score[relation[1]]++;
        }
        for(int person=1;person<=n;person++)if(score[person]==n-1)return person;
        return -1;
    }
}

File: content/problems/0997-find-the-town-judge/solutions/net-trust-degree/solution.cpp
class Solution {
    public: int findJudge(int n,vector<vector<int>>& trust){
        vector<int> score(n+1);
        for(auto& relation:trust){
            --score[relation[0]];
            ++score[relation[1]];
        }
        for(int person=1;person<=n;++person)if(score[person]==n-1)return person;
        return -1;
    }
};
